flintrock/flintrock.py

```diff
@@ -226,6 +226,8 @@ def cli(cli_context, config, provider):
 @click.option('--ec2-ebs-optimized/--no-ec2-ebs-optimized', default=False)
 @click.option('--ec2-instance-initiated-shutdown-behavior', default='stop',
               type=click.Choice(['stop', 'terminate']))
+@click.option('--ec2-user-data', default='',
```

Comment:
That's great. I missed that point. 
I would change it to Click's `Path` type and remove `_get_user_data()`.